src/ai/backend/test/cli_integration/user/test_vfolder.py

```diff
@@ -71,6 +73,123 @@ def test_rename_vfolder(run_user: ClientRunnerFunc):
     assert bool(test_folder3), "Test folder 3 doesn't exist!"
 
 
+def test_upload_file(run_user: ClientRunnerFunc, txt_file: TextIOWrapper):
```

Comment:
If execution order is crucial, you may use `pytest-dependency`. Please refer the code below or the [docs](https://pytest-dependency.readthedocs.io/en/stable/usage.html).
https://github.com/lablup/backend.ai/blob/9df3058fd9981e54737a5905f6e12b83e133c35c/tests/manager/models/test_etcd.py#L95-L97